src/modules/job-post/job-post.controller.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  Param,
+  Patch,
+  Post,
+  UseGuards,
+} from '@nestjs/common';
+import { JobPostService } from './job-post.service';
+import { CreateJobPostDto } from './dto/create-job-post.dto';
+import { JwtAuthGuard } from '../auth/guards/jwt-auth.guard';
```

Comment:
Should use absolute path 'src/modules/auth/guards/jwt-auth.guard'

Reply:
done